app/utils/rw_api.py

```diff
@@ -185,3 +195,210 @@ async def signup(name: str, email: str) -> User:
         )
 
     return User(**response.json()["data"])
+
+
+async def create_rw_geostore(
+    payload: RWGeostoreIn, x_api_key: str | None = None
+) -> RWGeostoreResponse:
+    url = f"{RW_API_URL}/v1/geostore"
```

Comment:
This should be `v2`, right?
```suggestion
    url = f"{RW_API_URL}/v2/geostore"
```

Reply:
> This should be `v2`, right?

Hmm, I've never tried with v2. The original ticket didn't say, and I got the v1 from https://resource-watch.github.io/doc-api/reference.html#overview-of-available-endpoints

I just tried, and they both work. Seemingly identical results.

Reply:
Let's stick with your current implementation, @dmannarino. We want bug-for-bug compatibility right now.

I looked at the code for the above `v1` endpoints and compared with `v2`. They are different.
But, I looked at Flagship's usage of the `create` endpoint. It looks like it just calls `/geostore` with no version and that seems to make the microservice default to `v1`. I could not find evidence of `calculate area` being called.

We'll need to reconcile the `v1` vs `v2` endpoints at a later date.

Reply:
Okay, thanks for checking that out!